paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -818,6 +818,13 @@ public class CoreOptions implements Serializable {
                                     + " the value should be the user configured local time zone. The option value is either a full name"
                                     + " such as 'America/Los_Angeles', or a custom timezone id such as 'GMT-08:00'.");
 
+    public static final ConfigOption<Boolean> MERGE_SCHEMA =
```

Comment:
Maybe we can introduce a `SparkConnectorOptions`? This is only for spark.

Reply:
I think so.